styles.css

```diff
@@ -35,6 +36,14 @@ a.txt-white:hover {
   width: 100%;
 }
 
+.help-info:target {
+  display: block;
+}
+
+.help-info:not(:target) {
+  display: none;
+}
```

Comment:
You can hide the map with the sibling selector:

.help-info:target + .map {
  display: none;
}

Reply:
That worked, huzzah!